project/Paradox.scala

```diff
@@ -72,7 +72,6 @@ object Paradox {
 
   val themeSettings = Seq(
     // allow access to snapshots for pekko-sbt-paradox
-    resolvers += Resolver.ApacheMavenSnapshotsRepo,
```

Comment:
Its cleaner to put this into root `build.sbt` because that way we can easily remove it once Pekko is released.

Reply:
Let me redo this change, pekko core doesn't really need it since its at the top of the dependency change

Reply:
So this dependency shouldn't even be here, pekko-sbt-paradox is an sbt plugin which means it gets resolved in `project/plugins.sbt`.

Reply:
(perhaps it is used to fetch themes?)

Reply:
I just checked and that doesn't appear to be the case https://github.com/apache/incubator-pekko-sbt-paradox/blob/main/build.sbt#L52

Reply:
Okay I was wrong, I missed `https://repo1.maven.org/maven2/org/apache/pekko/pekko-theme-paradox/0.0.0+38-68da3106-SNAPSHOT/pekko-theme-paradox-0.0.0+38-68da3106-SNAPSHOT.pom`. Reverting the change

Reply:
> There is a `resolverSettings ` in `PekkoBuild.scala`, would that be be better place?

This is temporary so actually I would prefer to not have it in `resolverSettings` which is also designed to be global (in our case we only want it for Paradox).